go test doesn't work

```bash
$ go test ./...
pkg/http/h2_bundle.go:47:2: use of vendored package golang_org/x/net/http/httpguts not allowed
pkg/http/transport.go:34:2: use of vendored package golang_org/x/net/http/httpproxy not allowed
pkg/http/h2_bundle.go:48:2: use of vendored package golang_org/x/net/http2/hpack not allowed
pkg/http/h2_bundle.go:49:2: use of vendored package golang_org/x/net/idna not allowed
pkg/http/httptest/recorder.go:15:2: use of vendored package golang_org/x/net/http/httpguts not allowed
pkg/http/httptest/server.go:17:2: use of internal package net/http/internal not allowed
pkg/http/httptrace/trace.go:12:2: use of internal package internal/nettrace not allowed
pkg/http/httputil/httputil.go:11:2: use of internal package net/http/internal not allowed

$ go test -mod=vendor ./...
pkg/http/h2_bundle.go:47:2: use of vendored package golang_org/x/net/http/httpguts not allowed
pkg/http/transport.go:34:2: use of vendored package golang_org/x/net/http/httpproxy not allowed
pkg/http/h2_bundle.go:48:2: use of vendored package golang_org/x/net/http2/hpack not allowed
pkg/http/h2_bundle.go:49:2: use of vendored package golang_org/x/net/idna not allowed
pkg/http/httptest/recorder.go:15:2: use of vendored package golang_org/x/net/http/httpguts not allowed
pkg/http/httptest/server.go:17:2: use of internal package net/http/internal not allowed
pkg/http/httptrace/trace.go:12:2: use of internal package internal/nettrace not allowed
pkg/http/httputil/httputil.go:11:2: use of internal package net/http/internal not allowed
```

--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,7 @@
 SHELL   := /bin/bash -euo pipefail
 TIMEOUT := 1s
 GOFLAGS := -mod=vendor
+PKGS    := go list ./... | grep -v pkg/http
 
 
 .PHONY: deps
@@ -19,7 +20,11 @@ format:
 
 .PHONY: test
 test:
-	@go test -race -timeout 1s ./...
+	@$(PKGS) | xargs -I {} go test -race -timeout $(TIMEOUT) {}
+
+.PHONY: test-with-coverage
+test-with-coverage:
+	@$(PKGS) | xargs -I {} sh -c "go test -cover -timeout $(TIMEOUT) {} | column -t | sort -r"
 
 
 .PHONY: build

